src/scitacean/testing/backend/seed.py

```diff
@@ -93,7 +93,7 @@
         techniques=[UploadTechnique(pid="S", name="shoes")],
     ),
     "partially-broken": model.construct(
-        UploadRawDataset,
+        UploadDerivedDataset,
```

Comment:
Why this change? Seems unrelated?

Reply:
The type was already 'derived' (see the `type` field). This change fixes a mistake that went under the radar because the old pydantic was more lenient with the types of models. TBH, I don't know why this code didn't raise an exception before but it did now before this change.